sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/Crashes.java

```diff
@@ -371,6 +438,9 @@ protected synchronized void applyEnabledState(boolean enabled) {
             /* Delete cache and in memory last session report. */
             mErrorReportCache.clear();
             mLastSessionErrorReport = null;
+            mContext.unregisterComponentCallbacks(mMemoryWarningListener);
```

Comment:
Does it work with null? As in starting disabled.

Reply:
yes